docs/install.rst

```diff
@@ -42,6 +42,13 @@ the cloned ``gala`` directory)::
 
     python -m pip install .
 
+For the best performance, you may wish to compile with architecture-specific optimizations::
+
+    export CXXFLAGS=-march=native
+
+This will likely have the biggest effect on orbit integration. Be aware that compiling with this
+flag means that Gala will only run on the same type of CPU that it was compiled on!
+
```

Comment:
Do we want to make this the default, and add a command-line option within setup.py to disable this? I guess then we would also need to make sure the wheels build with that disabled.

Reply:
I think you can get nearly all the performance with a more generic flag like `-march=x86-64-v3` (Haswell, so 12 years old). I would probably be inclined to use that by default instead of `-march=native`. The downside is that it requires adding logic to `setup.py` to detect whether the target CPU is x86. That's probably better than giving unsuspecting users "illegal instruction error" though...